pages/agent/v3/prioritization.md.erb

```diff
@@ -19,3 +19,11 @@ or with the `BUILDKITE_AGENT_PRIORITY` an environment variable:
 ```
 env BUILDKITE_AGENT_PRIORITY=9 buildkite-agent start
 ```
+
+## Load balancing
+
+You can use the Agent priority value to load balance jobs across Agents. 
+
+For example if you have 3 Agents on a machine, you would set one Agent to `priority=3`, one to `priority=2`, and one to `priority=1`. 
+
+Buildkite will then automatically load balance the jobs across the Agents, as it will assign jobs to the high priority agents first.
```

Comment:
```suggestion
Buildkite will then automatically load balance the jobs across the machines, as it will assign jobs to the high priority agents first.
```